core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -271,6 +271,18 @@ private boolean isInternalTopic(final String topicName) {
             && (topicName.endsWith("-changelog") || topicName.endsWith("-repartition"));
     }
 
+    private void printHelp(OptionParser parser) throws IOException {
+        System.err.println("The Application Reset Tool allows you to quickly reset an application in order to reprocess its data from scratch.\n" +
+                "* This tool will reset offsets of input topics to 0 and it skips to the end of intermediate topics (topics created with the through() method).\n" +
+                "* This tool will deletes the internal topics that were created automatically (topics ending in -changelog and -repartition).\n  You do not need to specify internal topics the tool finds them automatically.\n" +
+                "* This tool will not delete output topics (if you want to delete them, you need to do it yourself through bin/kafka-topics.sh command)\n" +
+                "* This tool will does not clean up the local state on the stream application instances (the persisted stores used to cache aggregation results).\n  You need to call KafkaStreams#cleanUp() in your application or manually delete them from the directory specified by state.dir configuration (usually /tmp/kafka-streams/<application.id>\n\n" +
```

Comment:
`usually` -> `default`
(I hope that people *usually* change the default to avoid unwanted state recreating if `temp` gets wiped out :)

And: closing `)` missing at the end.